api/server/router/system/system_routes.go

```diff
@@ -38,16 +38,34 @@ func (s *systemRouter) pingHandler(ctx context.Context, w http.ResponseWriter, r
 	}
 
 	w.Header().Set("Swarm", s.swarmStatus())
+	backendInfo, err := s.backend.SystemInfo(ctx)
```

Comment:
I think this calls the same code as is executed for `/info` ? 😬

If so, we should probably look at a better way to pass this information, otherwise `/_ping` (intended to be as lightweight as possible) becomes as heavyweight as `/info` (which is exactly the endpoint we try to avoid for these)